src/App.js

```diff
@@ -23,6 +24,7 @@ const App = () => (
           <Route exact path="/dashboard" component={Authenticate(SourceSearch)} />
           <Route exact path="/dashboard/concepts" component={Authenticate(ConceptSearch)} />
           <Route exact path="/dashboard/dictionaries" component={Authenticate(DictionaryDisplay)} />
+          <Route exact path="/dashboard/concepts/:organisation/:name" component={Authenticate(SpecificConcept)} />
```

Comment:
is it organisation or organization?

Reply:
Organisation and organization mean the same thing.
They are alternative spellings of the same word, both being used in British English and only one being used in American English.
1. Both words can be used in British English.
2. Organization is the only correct spelling in American English.

Reply:
my suggestion would be to go with what is currently in use the one with a 'Z', slight error but might be confusing for one who needs to write a function for the organizations module

Reply:
I now understand what you meant. Let me make the changes.

Reply:
I have made the changes